#!/bin/sh /usr/share/dpatch/dpatch-run

## 01_Makefile.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Move installation of README and man page from all to install.

@DPATCH@
--- osdteletext.orig/Makefile
+++ osdteletext/Makefile
@@ -67,8 +67,6 @@
 ### Targets:
 
 all: libvdr-$(PLUGIN).so
-	@install -d ../../man
-	@install README ../../man/$(PLUGIN).man
 
 libvdr-$(PLUGIN).so: $(OBJS)
 	$(CXX) $(CXXFLAGS) -shared $(OBJS) -o $@
@@ -84,3 +82,7 @@
 
 clean:
 	@-rm -f $(OBJS) $(DEPFILE) *.so *.tgz core* *~
+
+install:
+	@install -d ../../man
+	@install README ../../man/$(PLUGIN).man
